src/odemis/acq/test/move_test.py

```diff
@@ -281,30 +287,101 @@ def test_smaract_stage_fallback_movement(self):
         cryoSwitchSamplePosition(THREE_BEAMS).result()
         # 2. Move the stage linear axes to their max range + move rx from 0
         cryoSwitchAlignPosition(LOADING).result()
-        self.stage.moveAbs({'x': self.stage.axes['x'].range[1], 'y': self.stage.axes['y'].range[1], 'z': self.stage.axes['z'].range[1], 'rx': 0.15}).result()
+        self.stage.moveAbs(
+            {'x': self.stage.axes['x'].range[1], 'y': self.stage.axes['y'].range[1], 'z': self.stage.axes['z'].range[1],
+             'rx': 0.15}).result()
         # 3. Move to loading where the ordered submoves would start from rx/rx, resulting in an invalid move
         # exception if it's not handled
         cryoSwitchSamplePosition(LOADING).result()
         testing.assert_pos_almost_equal(self.stage.position.value, self.stage_deactive,
-                                     atol=ATOL_LINEAR_POS)
+                                        atol=ATOL_LINEAR_POS)
 
 
 class TestMeteorMove(unittest.TestCase):
     """
     Test the function cryoSwitchSamplePosition stage movements of meteor 
     """
+
     @classmethod
     def setUpClass(cls):
-        testing.start_backend(METEOR_CONFIG)
+        # Check the linked YM plane which is at a pre-tit angle beta of 26-degrees using GUI simulation
+        # to cover all the test cases
+
+        testing.start_backend(METEOR_ZEISS_CONFIG)
 
         # get the stage components
         cls.stage = model.getComponent(role="stage-bare")
+        cls.linked_ym_stage = model.getComponent(role="stage")
 
         # get the metadata
         stage_md = cls.stage.getMetadata()
         cls.stage_grid_centers = stage_md[model.MD_SAMPLE_CENTERS]
         cls.stage_loading = stage_md[model.MD_FAV_POS_DEACTIVE]
 
+    def test_transformFromSEMToMeteor(self):
```

Comment:
Please add a test that takes a SEM position, and convert it to FM, and convert it back to SEM. That final position should be the same as the original position. You can easily try with many positions.
In some way it's a weaker test than the two current tests, but first, it checks that the two functions are opposite, and secondly, it's less "brittle". Even if you change a bit the formula, you don't need to update the test case.